src/express/commands/cleanup.js

```diff
@@ -64,8 +64,8 @@ async function cleanupServices(doc) {
     }
 
     let cleanupServicesTasks = nodeIps.map(async(ip) => {
-        let command = `sudo mkdir -p /var/lib/heimdall && sudo chmod 777 -R /var/lib/heimdall/ `
-        await runSshCommand(ip, command, maxRetries)
+        //let command = `sudo mkdir -p /var/lib/heimdall && sudo chmod 777 -R /var/lib/heimdall/ `
```

Comment:
can we remove the comments if not needed?

Reply:
[7e27cdf](https://github.com/maticnetwork/matic-cli/pull/91/commits/7e27cdf2e41b2a3f04dc3eef1217c4a73af982f7)